Tasks/Xcode/task.json

```diff
@@ -226,6 +231,55 @@
             "visibleRule": "signStyle = auto"
         },
         {
+            "name": "platform",
+            "type": "picklist",
+            "label": "Destination platform",
+            "defaultValue": "default",
+            "required": false,
+            "helpMarkDown": "Select the destination device's platform. If you need an option not listed below, use `Arguments` to pass the -destination argument.",
+            "groupName": "devices",
+            "options": {
+                "default": "default",
+                "iOS": "iOS and watchOS",
+                "tvOS": "tvOS",
+                "macOS": "macOS"
+            }
+        },
+        {
+            "name": "destinationType",
```

Comment:
testDestinationType to be more YAML friendly?

Reply:
`destinationTypeOption`

Reply:
went with destinationPlatformOption, destinationPlatform and destinationTypeOption to get some stacking and hopefully intellisense in yaml.

Reply:
went ahead and stacked devices (`destinationDevices`) and simulators (`destinationSimulators`) too to make it clear to yaml users that all the settings are related.